src/gt4sd/frameworks/molgx/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
all these files are not needed as authors, license, etc. are covered by the gt4sd ones.